### PR TITLE
Replaces security armor with new, modern versions.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
rscadd: adds new versions of Security and Blueshield armors. They're now standard NT gear.
/:cl:

New Security armor and helmet and new Bluecurity armor

NOBODY READS THIS
IF YOU READ THIS, PM ME, I'LL GIVE YOU A COOKIE

Also, sprites with a mask on, because why not
BOOP BEEP

The stats and abilities remain unchanged.

All officers will start with this version of the armor and helmet, as well as having those in the lockers. The old one is now only obtainable through admin intervention.

I'm a crappy spriter, everyone knows that, so send me better versions of the sprites if you think you can do better.

Also, the helmets are totally ripped from /tg/. No regrets.

### CODE REVIEW

**Comment by tigercat2000** [comment]
don't like it, our current one is fine

**Comment by NTSAM** [comment]
but our current one is bland and undetailed

**Comment by LiquidNightfury** [comment]
Love it. No more fat body armor.

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -17,6 +17,9 @@
 		"Drask" = 'icons/mob/species/drask/helmet.dmi'
 		)
 
+/obj/item/clothing/head/helmet/modern
```
Comment (tigercat2000) [inline code/modules/clothing/head/helmet.dm]: you really shouldn't make subtypes if you don't need to, there is a theoretical max on typepaths
just change the sprite

Comment (CrAzYPiLoT-SS13) [inline code/modules/clothing/head/helmet.dm]: Out of curiosity, what would be the max?

**Comment by Iamgoofball** [comment]
eeeugh

these look fucking horrifying

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (tigercat2000) [inline code/modules/clothing/head/helmet.dm]: @CrAzYPiLoT-SS13 I don't recall the exact details, but I am pretty sure there's a hard limit somewhere over number of different paths you can have

**Comment by Vivalas** [comment]
Not a fan of these sprites. The current ones look fine, these are way too dark and the gold badge isn't supposed to be on security armor, it's for the detective,

**Comment by sasanek12** [comment]
:+1: For any new sprite for armour and helmet, current ones are horrible in my opinion.

**Comment by Fruerlund** [comment]
👍

Looks very good. Current ones are plain and boring. I don't think these are too dark either.
If it's an issue with the badge being for the detective then change them to look like the badges from the Wardens Locker.

**Comment by KasparoVy** [comment]
TG had a war over this sort of thing for a very long time, and they've got some really really nice sprites in the mix. Why not take a peek at those?

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: 65536, but I believe types were one of the limited things in DM that were uncapped to "functionally infinite"

Comment (tigercat2000) [inline code/modules/clothing/head/helmet.dm]: oh i guess it doesn't matter then

**Comment by MrBarrelrolll** [comment]
I like it a lot. Badge should be changed so as not to conflict with the detective, but the ones in the armory would be a good replacement. Badge might be a bit big? But I think it's alright.

**Comment by FlattestGuitar** [comment]
Okay, considering some people dislike the sprites and we all deep at heart want a crapton more of those, I'll wait for @tigercat2000's [#4417](https://github.com/ParadiseSS13/Paradise/pull/4417) to get merged and I'll bother with new sprites then. I'll prolly end up stealing a lot of stuff from /tg/, because they have a nice amount of those stored in their numerous closed PRs. 
Cheers.

**Pull request closed without merging**